Create secondary subject page (#4363)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary.md

---
title: Become a secondary school teacher 
subject: secondary
title_paragraph: |-
  <p>

>   As a secondary school teacher, you'll play a pivotal role in pupils' lives. You'll see them grow from children to young adults, inspiring them to become passionate about learning and helping shape their futures.</p>

Reviewer: ```suggestion
  As a secondary school teacher, you'll play a pivotal role in pupils' lives. You'll see them grow from children to young adults, inspiring them to become passionate about learning and help shape their futures.</p>
```

---
title: Become a secondary teacher 
subject: secondary
title_paragraph: |-
  <p>

>   As a secondary teacher, you'll play a pivotal role in students' lives. You'll see them grow from children to young adults, inspiring them to become passionate about your subject and shaping their futures.</p>

Reviewer: ```suggestion
  As a secondary teacher, you'll play a pivotal role in pupils' lives. You'll see them grow from children to young adults, inspiring them to become passionate about learning and shaping their futures.</p>
```

  
description: |-

>     Find out how to become a secondary teacher, including what you'll teach and what funding is available to help you train.

Reviewer: Not sure whether it's officially secondary teacher or secondary school teacher, but I prefer secondary school teacher, as secondary teacher sounds like a second choice almost?

layout: "layouts/minimal"
colour: pastel yellow-yellow
image: "static/images/content/hero-images/chemistry.jpg"
keywords:
  - secondary

>   - secondary teaching

Reviewer: ```suggestion
  - secondary teaching
  - secondary school teacher
```

>   - secondary teacher training

Reviewer: ```suggestion
  - secondary teacher training
  - secondary school teacher training
```
